oarepo_dashboard/ui/dashboard_components/semantic-ui/js/dashboard_components/search/FacetsButtonGroupNameToggler.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext } from "react";
+import PropTypes from "prop-types";
+import { i18next } from "@translations/oarepo_dashboard";
+import { Button } from "semantic-ui-react";
+import { withState } from "react-searchkit";
+import { SearchConfigurationContext } from "@js/invenio_search_ui/components";
+
+const FacetsButtonGroupNameTogglerComponent = ({
+  currentResultsState,
+  currentQueryState,
+  updateQueryState,
+  facetNames,
+  firstFacetButtonText,
```

Comment:
This needs to be a bit more flexible, not just 2 buttons (refactored into something like `facetOptions` prop). We can deal with that in #FE-141

Reply:
@mirekys Clear! thanks